CogniteSdk.Types/Alpha/Simulators/SimulatorRoutineRevisionIO.cs

```diff
@@ -0,0 +1,97 @@
+// Copyright 2024 Cognite AS
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Text.Json.Serialization;
+using CogniteSdk.Types.Common;
+
+namespace CogniteSdk.Alpha
+{
+    /// <summary>
+    /// Simulator routine revision input.
+    /// </summary>
+    public class SimulatorRoutineRevisionInput {
+        /// <summary>
+        /// Name of the input.
+        /// </summary>
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Reference id of the input.
+        /// </summary>
+        public string ReferenceId { get; set; }
+
+        /// <summary>
+        /// If defined, this is time series external ID the value will be read from.
+        /// </summary>
+        public string SourceExternalId { get; set; }
+
+        /// <summary>
+        /// Value of the input. Only used if the input is not a time series.
+        /// </summary>
+        public SimulatorValue Value { get; set; }
+
+        /// <summary>
+        /// Value type of the input.
+        /// </summary>
+        public SimulatorValueType ValueType { get; set; }
+
+        /// <summary>
+        /// Unit of the input.
+        /// </summary>
+        public SimulatorValueUnit Unit { get; set; }
+
+        /// <summary>
+        /// If defined, this is time series external ID the value will be persisted to.
+        /// </summary>
+        public string SaveTimeseriesExternalId { get; set; }
+
+        /// <inheritdoc />
+        public override string ToString() => Stringable.ToString(this);
+
+        /// <summary>
+        /// Whether the input is a time series.
+        /// </summary>
+        [JsonIgnore]
```

Comment:
what does the [JsonIgnore] decorator do?

Reply:
this is the calculated property, we can use it for convenience in the code, but we shouldn't serialise it once calling the API. this prevents it being output into json

Reply:
nice